content/templates/layouts/default/page.html

```diff
@@ -53,11 +53,19 @@
           <div class="collapse navbar-collapse" id="navbarsExampleDefault">
             <ul class="navbar-nav justify-content-between" style="width: 100%">    
           <li class="nav-item dropdown">
+<<<<<<< HEAD
```

Comment:
This shows a conflict in the file when you merged it.
You need to resolve the conflict by choosing what you actually want in the file between this line and 
">>>>>>> 26cf4f7 (Squashed two last commits into one)"

Reply:
@simzacks  I have fixed it.